### PR TITLE
[ARTEMIS] New_default tests removed

### PR DESCRIPTION
**Pull request by AzHicham**
New default tests are deprecated, and therefore removed in Artemis.

### CODE REVIEW

**Comment by AzHicham** [comment]
This PR is still under testing in Jenkins. 
Merge this PR after passing Jenkins [Artemis NG]

**Review by kadhikari — COMMENTED** [review]
I think you can also delete "new_default" in:
* https://github.com/CanalTP/artemis/blob/0c2e14d335e14d563eb778b96786265aa1c14b30/artemis/instance_default_values.py#L2
* https://github.com/CanalTP/artemis/blob/7cb8e3282d479aefb1785aa6797af3802290c6ed/artemis/base_pytest.py#L496

File: artemis/tests/airport01_test.py [hunk 1]
```diff
@@ -41,11 +41,6 @@ def test_airport_01_04(self):
         )
 
 
-@set_scenario({"airport-01": {"scenario": "new_default"}})
-class TestAirport1NewDefault(Airport1, ArtemisTestFixture):
-    pass
-
-
 @set_scenario({"airport-01": {"scenario": "experimental"}})
```
Comment (pbench) [inline artemis/tests/airport01_test.py:44]: Is it easy to also remove these annotations ? And the rest of the code related to the "scenario" feature ?

**Review by pbench — APPROVED** [review]
:tada:

**Review by kadhikari — APPROVED** [review]
I suppose scenario default value is 'distributed'
:+1:

**Comment by AzHicham** [comment]
PR Test with : 
- [Artemis NG ](https://jenkins-core.canaltp.fr/job/artemis_ng/243/)
- [Artemis Old ](https://jenkins-core.canaltp.fr/job/artemis_old/189/)
NB : Tested with the following [PR in Artemis_REF ](https://github.com/CanalTP/artemis_references/pull/239)

File: artemis/test_mechanism.py [hunk 2]
```diff
@@ -233,7 +232,7 @@ def clean_jormun_db(cls):
             for data_set in cls.data_sets:
                 cur.execute(
                     "INSERT INTO instance (name, is_free, is_open_data, scenario) VALUES ('{}', true, false, '{}');".format(
-                        data_set.name, data_set.scenario
+                        data_set.name, "experimental"
```
Comment (xlqian) [inline artemis/test_mechanism.py:235]: can you change the scenario's name to `distributed` by the way plz?

**Review by xlqian — APPROVED** [review]

File: artemis/test_mechanism.py (hunk 2 above)
Comment (AzHicham) [reply artemis/test_mechanism.py:235]: Ok but will it still works in Artemis Old ?

Comment (AzHicham) [reply artemis/test_mechanism.py]: Done !

**Pull request merged**